src/coreclr/src/jit/instr.cpp

```diff
@@ -1086,8 +1089,161 @@ void CodeGen::inst_RV_TT_IV(instruction ins, emitAttr attr, regNumber reg1, GenT
         GetEmitter()->emitIns_SIMD_R_R_I(ins, attr, reg1, rmOpReg, ival);
     }
 }
+
+//------------------------------------------------------------------------
+// inst_RV_RV_TT: Generates an instruction that takes 2 operands:
+//                a register operand and an operand that may be in memory or register
+//                the result is returned in register
+//
+// Arguments:
+//    ins       -- The instruction being emitted
+//    size      -- The emit size attribute
+//    targetReg -- The target register
+//    op1Reg    -- The first operand register
+//    op2       -- The second operand, which may be a memory node or a node producing a register
+//    isRMW     -- true if the instruction is RMW; otherwise, false
+//
+void CodeGen::inst_RV_RV_TT(
+    instruction ins, emitAttr size, regNumber targetReg, regNumber op1Reg, GenTree* op2, bool isRMW)
```

Comment:
`isRMW` should come before `targetReg` & operands. It's already pretty dubious that such information has to be communicated separately from instructions.

Reply:
I put this at the end to keep parameter ordering with other overloads consistent and because this should likely be looked up a different way eventually (either based on the instruction or the `node`)